samples/net/socket_echo/prj_qemu_x86.conf

```diff
@@ -0,0 +1,26 @@
+# General config
+CONFIG_NEWLIB_LIBC=y
+
+# Networking config
+CONFIG_NETWORKING=y
+CONFIG_NET_IPV4=y
+CONFIG_NET_IPV6=n
+CONFIG_NET_TCP=y
+CONFIG_NET_SOCKETS=y
+CONFIG_NET_SOCKETS_POSIX_NAMES=y
+
+# Network driver config
+CONFIG_NET_SLIP_TAP=y
+CONFIG_TEST_RANDOM_GENERATOR=y
+
+# Without CONFIG_NET_BUF_LOG printf() doesn't work
+CONFIG_NET_BUF_LOG=y
```

Comment:
Normally one does not need to enable this unless one is debugging the actual net_buf implementation.